simcon_project/templates/static/researcherTable.css

```diff
@@ -0,0 +1,28 @@
+.body-container {
```

Comment:
again, this file is deleted. please slack me and we can fix ur git state. this file was moved